tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -1034,11 +1034,11 @@ static_assert(test_case_iterators<char, constexpr_char_traits>());
 static_assert(test_case_prefix<char, constexpr_char_traits>());
 static_assert(test_case_suffix<char, constexpr_char_traits>());
 static_assert(test_case_swap<char, constexpr_char_traits>());
-static_assert(test_case_copy<constexpr_char_traits>());
 static_assert(test_case_Copy_s<constexpr_char_traits>());
 static_assert(test_case_substr<constexpr_char_traits>());
 static_assert(test_case_compare<char, constexpr_char_traits>());
 #if _HAS_CXX20
+static_assert(test_case_copy<constexpr_char_traits>());

```

Comment:
If we revert the change to `basic_string_view` we should revert this test change also.